source/_components/alarm_control_panel.paradox.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: page
+title: "Paradox Alarm Control Panel"
+description: "Instructions how to integrate Paradox alarms into Home Assistant."
+date: 2016-10-07 12:17
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: paradox.png
+ha_category: Alarm
+ha_release: 0.30
+ha_iot_class: "Local Push"
+---
+
+The `paradox` alarm control platform is a sub module of the [Paradox hub](/components/paradox/) which provides integration with Paradox alarms utilizing the PRT3 module.
+
+Setting up the hub allows you to enable this platform, which exposes all of your areas/partitions as alarm control panels. Doing so provides visibility of the areas through the UI as well as the ability to trigger automation actions when something happens like an area being armed/disarmed.
+
+The PRT3 module have no way to indicate which areas/partitions are actually in use. It has to be controlled via the `configuration.yaml` file. Add only the areas/partitions you wish to be visible in the UI. Any areas not specified will simply be ignored.
+
+An example `configuration.yaml` file, making area 1 and 2 visible in the UI will look something like this:
+
+
+```yaml
+# Example Paradox configuration.yaml entry
+paradox:
+  panel_type: EVO48
+  port: /dev/ttyUSB0
+  speed: 57600
+
+  partitions:
+    1:
+      name: 'Home'
+    2:
+      name: 'Backyard'
+```
+
+Configuration variables:
+
+- **hub entries** (*Required*): See [Paradox hub](/components/paradox/) for details.
+- **[partition]number** (*Required*): This number must match the area number in use on the alarm panel.
+- **[partition]name** (*Required*): Use this name to specify a name for the area/partition as you want to see it displayed in Home Assistant. It does not need to match the name of the area in the alarm panel.
```

Comment:
Thanks for all the feedback!

On this one I added the links so as not to have the same information on three different places and thus running the risk of one of the three being out of date after an update.

I also wasn't sure whether I should leave the partitions and zone out of the example yaml file in the hub again so as to remove the risk out out of date information.